### PR TITLE
Display list items

### PR DESCRIPTION
**Pull request by ookino**
## Features Added ♻️ 
-Added function to get meals from 'The meals' API and render them

### CODE REVIEW

File: src/modules/render.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    p.className = 'meal-title';
+    p.innerText = mealItem.strMeal;
+    div.appendChild(p);
+    const likeBtn = document.createElement('button');
+    likeBtn.className = 'icn-btn';
+    div.appendChild(likeBtn);
+    const likeIcon = document.createElement('i');
+    likeIcon.className = 'far fa-heart';
+    likeBtn.appendChild(likeIcon);
+    const commentBtn = document.createElement('button');
```
Comment (lily-coder) [inline src/modules/render.js:27]: Great work on implementing the Comments button, but I think you forgot to add the reservations button too

'Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.'

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hello @yaseerokino ,
Great job so far!
You have been doing an incredible job on the project so far! I love it👏👏
The project is very well structured😃😃
However, we still need to make a few changes💪

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: src/modules/render.js (hunk 1 above)
Comment (ookino) [reply src/modules/render.js:27]: Resolved

**Review by lily-coder — APPROVED** [review]
### Approved👏
Hi,
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

**Pull request merged**